resources/assets/components/CampaignSingle/index.js

```diff
@@ -34,9 +23,48 @@ class CampaignSingle extends React.Component {
     this.filterPosts = this.filterPosts.bind(this);
   }
 
+  componentDidMount() {
+    this.api.get('api/v2/posts', {
+      filter: {
+        status: 'accepted',
+        campaign_id: this.props.campaign.id,
+      },
+      include: 'signup,siblings',
```

Comment:
You're right, it doesn't work yet :( But I agree - that looks a lot better! Should I great an issue for that in Gateway?